randomArray.js

```diff
@@ -0,0 +1,17 @@
+const _ = require("underscore")
+
+function mixedArray1 (arr) {
+  const newArr = _.shuffle(arr);
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] === newArr[i]){
+      newArr.splice(i, 1);
+      const randomIndex = Math.floor(Math.random() * newArr.length);
```

Comment:
Hi @yjlim5,
Thank you for reviewing my code!
I will look at it tonight and apply the element swapping to the code instead of using splice.